src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -11,15 +11,20 @@
 
 namespace Symfony\Bundle\WebProfilerBundle\Controller;
 
+use Symfony\Bundle\FrameworkBundle\Client;
```

Comment:
this adds dependencies on the FrameworkBundle, while there is no such dependency currently (see usage in Twig for instance)